lib/facility_factory.rb

```diff
@@ -0,0 +1,22 @@
+class FacilityFactory
+    
+    def create_facilities(facility_details)
+        
+        facilities_created = []
+        facility_details.each do |facility|
+          facility1 = {}
```

Comment:
I might call this something like `facility_data`, `facility1` feels arbitrary and less dynamic than it actually is.